tests/ui/specs/Scroller/ListOfThings/Scroller-GT-28586-specs.js

```diff
@@ -0,0 +1,26 @@
+const ScrollerPage = require('../ScrollerPage');
+
+describe('Scroller List Of Things', function () {
+	beforeEach(function () {
+		ScrollerPage.open('ListOfThings');
+	});
+
+	it('should spotlight on x closing button with spotlight disabled [GT-28586]', function () {
+		// Step 3: Knobs > Scroller > spotlightDisabled > check
```

Comment:
I got error on this TC
```

[0-5] Error in "Scroller List Of Things should spotlight on x closing button with spotlight disabled [GT-28586]"
Error: unknown error: Element is not clickable at point (960, 2876)
  (Session info: headless chrome=88.0.4324.96)
  (Driver info: chromedriver=2.44.609551 (5d576e9a44fe4c5b6a07e568f1ebc753f1214634),platform=Linux 4.15.0-133-generic x86_64)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at Context.<anonymous> (/home/jeonghee27.ahn/Git/sandstone-new/tests/ui/specs/Scroller/ListOfThings/Scroller-GT-28586-specs.js:19:16)
[0-5] FAILED in chrome - /tests/ui/specs/Scroller/ListOfThings/Scroller-GT-28586-specs.js
```

Reply:
Sorry, I missed this fail. I fixed error.